src/main/java/core/selenium/WebDriverFactory.java

```diff
@@ -3,17 +3,29 @@
 import core.selenium.webdrivers.Chrome;
 import core.selenium.webdrivers.Firefox;
 import core.selenium.webdrivers.BrowserType;
+import core.selenium.webdrivers.IBrowser;
 import org.openqa.selenium.WebDriver;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import static core.selenium.webdrivers.BrowserType.CHROME;
+import static core.selenium.webdrivers.BrowserType.FIREFOX;
+
 /**
  * WebDriverFactory class, this class initializes a browser according to the type it receives.
  *
  * @author Maday Alcala
  * @version 0.0.1
  */
 public final class WebDriverFactory {
-    private static WebDriver webDriver;
-    private static final String MESSAGE_FOR_UNKNOWN_BROWSER = "Unknown browser type";
+    private static Map<BrowserType, IBrowser> map;
```

Comment:
Please you most be become variable of method.

Reply:
Solved.